### DISCUSSION TITLE
first launch surprise

### DISCUSSION THREAD

**Discussion by WojciechGw** [picocomputer/community#52] [Q&A] [answered]
I saw this prompt the first time I started the reference build of Picocomputer
![1698943894308](https://github.com/picocomputer/discussions/assets/7420761/7235dc40-9238-4605-9fe2-e32964dd0216)
Is it normal ? Where is magic "Picocomputer 6502 COLOR" ?
I used standard uf2 for Pico VGA and Pico RIA.

status and set lists
![1698948194698](https://github.com/picocomputer/discussions/assets/7420761/93556901-1e48-41c6-8ffa-a564eb2bbb1f)

test.rp6502 at work
![1698948194684](https://github.com/picocomputer/discussions/assets/7420761/4ba01159-5336-492a-870b-e53e19471ab8)

stable at 8Mhz

Kind regards

**Comment by rumbledethumps** [comment]
Surprise! 😲

**Comment by WojciechGw** [comment]
I've found 0.3 release, installed and ... nothing changed

![1699070306783](https://github.com/picocomputer/discussions/assets/7420761/e3b1bd68-e8ec-4feb-808f-184b2be60111)
![1699070384676](https://github.com/picocomputer/discussions/assets/7420761/ab546349-11b1-4707-a3a4-68d76883a461)

Do You have any ideas ?

BTW with my USB HUB, I have to connect USB devices after powering up the picocomputer, every time after shutting down I have to remove all USB devices from the hub before the next startup

**Reply by rumbledethumps** [reply]
USB issues come from TinyUSB which comes from the Pi Pico SDK. Most hubs work. Make sure it's not a specific device, then try a new hub.

**Reply by WojciechGw** [reply]
So far I have tried Unitek Hub H1108A with Unitek C438GBK USB-Micro USB OTG adapter - it didn't work (with rp6502).
Now I am using Aceele Hub AC-T10 with the same OTG adapter - it works except for what I mentioned earlier.

**Reply by WojciechGw** [reply]
Last time I used flash_nuke.uf2 on each pico before loading vga and ria uf2s firmware (release 0.3)

**Reply by rumbledethumps** [reply]
Even if I had a hub that didn't work, I don't know where to begin fixing it. You'll have to either find a working hub or wait for a new TinyUSB to come down stream with Pico SDK 1.5.2 and hope that fixes it.

**Reply by WojciechGw** [reply]
thank You for this information, but my primary question was about initial screen look :) any ideas ?

**Reply by rumbledethumps** [reply]
Looks the same as mine. What was the question?

**Reply by WojciechGw** [reply]
Where is magic "Picocomputer 6502 COLOR" :) as I saw on pictures from other users and Your videos ?

**Reply by rumbledethumps** [reply, marked as answer]
This project is under active development. I changed it.

**Reply by WojciechGw** [reply]
ok, now it's clear, thank You very much :) have a nice day :)
